### DISCUSSION TITLE
Wondering if anyone has been able to create a trakt list using their Plex collection?

### DISCUSSION THREAD

**Discussion by PlantationCane** [Taxel/PlexTraktSync#1815] [Q&A] [answered]
I would love to turn some of my lengthy plex collections into trakt lists using this tool.  I have PlexTraktsync working where it tracks my watched shows.  I would love to go one step beyond with the collections.

**Comment by glensc** [comment, marked as answer]
this app does not support syncing to trakt lists. you need to open feature request issue.
